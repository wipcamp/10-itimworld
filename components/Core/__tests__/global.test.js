/* eslint-env jest */

// import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import { Layout } from '../Global'

describe('<Layout />', () => {
  it('Layout should be renders', () => {
    const component = renderer.create(<Layout />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
