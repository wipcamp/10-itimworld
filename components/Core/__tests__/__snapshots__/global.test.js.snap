// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> Layout should be renders 1`] = `
<div
  className="sc-bdVaJa eAwGjB"
/>
`;
